### PR TITLE
Update to PSPF limitations in labelling

### PR DESCRIPTION
**Pull request by oobedan**
Updates include:

* PSPF M365 native limitations and possible resolutions
* Renamed AIP to MIP where appropriate

### CODE REVIEW

File: blueprint/office-365.md [hunk 1]
```diff
@@ -1233,27 +1233,35 @@ Alert Policies Status | Default Alert Policies enabled | Custom alert policies a
...
+The following methods can be used to assist with PSPF compliance when using MIP:
 
-Email gateway rules are available in the Network Configuration ABAC document. These rules are based on regular expressions and are easily adaptable to vendor specific email gateways.
+* Prepend the protective marking to the subject field using Exchange Online transport rules and MIP header checks.
+* For calendar invites, advise users to add the classification to the body, and use transport rules to add the required subject and header values.
+* Add the  `X-Protective-Marking` header associated with the MIP label at the email gateway, as transport rules cannot not populate the user's email address in the `ORIGIN=user@agency.gov.au` property inside the `X-Protective-Marking` header.
+
+Email gateway rules are available in the [Network Configuration ABAC document](https://www.desktop.gov.au/blueprint/abac/hybrid-network-configuration.html). These rules are based on regular expressions and are easily adaptable to vendor specific email gateways.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1264]: Is this referring to https://www.desktop.gov.au/blueprint/abac/hybrid-network-configuration.html#office-365--email-protective-markings-with-aip-technology ? Do references to "AIP" need to be updated to "MIP" too in the destination link?

File: blueprint/office-365.md [hunk 2]
```diff
@@ -1233,27 +1233,35 @@ Alert Policies Status | Default Alert Policies enabled | Custom alert policies a
...
+- When downgrading a Sensitivity label, the downgrade cannot be prevented, only forcing the user to justify the downgrade. 
+- Sensitivity labels are not available within calendar invites.
 
-Sensitivity labels create a protective marking within the message header and when combined with Exchange mail flow rules the subject can be modified to prepend text in the subject according to the sensitivity.
+The following methods can be used to assist with PSPF compliance when using MIP:
 
-Email gateway rules are available in the Network Configuration ABAC document. These rules are based on regular expressions and are easily adaptable to vendor specific email gateways.
+* Prepend the protective marking to the subject field using Exchange Online transport rules and MIP header checks.
+* For calendar invites, advise users to add the classification to the body, and use transport rules to add the required subject and header values.
+* Add the  `X-Protective-Marking` header associated with the MIP label at the email gateway, as transport rules cannot not populate the user's email address in the `ORIGIN=user@agency.gov.au` property inside the `X-Protective-Marking` header.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1262]: `as transport rules cannot not populate`

Is `cannot not` correct?

Comment (oobedan) [reply blueprint/office-365.md]: fixed

File: blueprint/office-365.md (hunk 1 above)
Comment (oobedan) [reply blueprint/office-365.md:1264]: yeah sorry should have looked in the other docs. Have replaced the other AIPs in the Platform design too now.

**Comment by zedzedzed** [comment]
If you do a global search for "AIP", there are some other files that may need to be updated. I'm not sure if AIP and MIP are easily interchangable so will leave that with you. The files I saw were overview, abac/office-365, continuous monitoring plan and abbreviations-acronyms.

**Comment by oobedan** [comment]
@zedzedzed found the updates and updated, as AIP is technically still a hangover product with the unified labelling client, I've explained in the Overview but there a few references still in places, as we actually use MIP in the blueprint.

**Review by zedzedzed — APPROVED** [review]

**Pull request merged**